source/containers.tex

```diff
@@ -80,6 +80,24 @@
 \tcode{r} denotes a non-const value of type \tcode{X}, and
 \tcode{rv} denotes a non-const rvalue of type \tcode{X}.
 
+% Local command to index names as members of all containers.
+\newcommand{\indexcont}[1]{%
```

Comment:
It's a little bit brittle to have one "newcommand" and several "renewcommand"s; that could easily be missed. Maybe make them all "renewcommand", and add an intial (error-causing?) placeholder "newcommand"?

Reply:
I understand the concerns about symmetry here, but what can possibly go wrong? If someone uses the macro before it's defined -> error. If someone uses the macro in a bad context -> tough luck.

Neither situation is helped with "renewcommand".

Reply:
Hm, I was concerned about silent changes of behaviour when we move blocks of text around, but you're right that renewcommand wouldn't help with that.